Dockerfile

```diff
@@ -1,41 +1,28 @@
-FROM centos:6
+FROM ruby:2.3-slim-jessie
 
 ARG sidekiq_license
 ENV BUNDLE_ENTERPRISE__CONTRIBSYS__COM=$sidekiq_license
 ARG exclude_sidekiq_ent
 ENV EXCLUDE_SIDEKIQ_ENTERPRISE=$exclude_sidekiq_ent
+ENV APP_PATH /src/vets-api
 
 # Match the jenkins uid/gid on the host (504)
 RUN groupadd -r vets-api && \
       useradd -r -g vets-api vets-api
 
-RUN yum install -y git make gcc-c++ openssl-devel readline-devel zlib-devel sqlite-devel postgresql-devel socat timeout epel-release nc
+RUN apt-get update -qq && apt-get install -y build-essential \
+		                             git \
+                                             libpq-dev \
+                                             nodejs \
+                                             clamav \
+                                             imagemagick \
+                                             pdftk
 
-# Install Red Hat SCI library for ruby packages
-RUN yum install -y centos-release-scl-rh
-RUN yum install -y rh-ruby23 rh-ruby23-ruby-devel rh-ruby23-rubygems
-RUN echo "source /opt/rh/rh-ruby23/enable" > /etc/profile.d/rh-ruby23.sh
-RUN ["/bin/bash", "--login", "-c", "gem install --no-doc bundler"]
+WORKDIR $APP_PATH
+ADD Gemfile $APP_PATH
+ADD Gemfile.lock $APP_PATH
+RUN bundle install
 
-# Install pdftk binary dependencies (see ansible/roles/pdftk/README.md)
-RUN yum install -y libgcj
-RUN curl https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/pdftk-2.02-1.el6.x86_64.rpm > /tmp/pdftk-2.02-1.el6.x86_64.rpm && \
-      rpm -i /tmp/pdftk-2.02-1.el6.x86_64.rpm && \
-      rm /tmp/pdftk-2.02-1.el6.x86_64.rpm
+COPY . APP_PATH
```

Comment:
I've been wondering about this since I did similar work on vets-website. Do we actually need to copy the files into the image? If we can just use a volume, it'd speed up the processes in the container significantly. I haven't thought through whether or not we need them copied over. There's no publishing of images or even copying of images that I know of in CI.